frontend/dashboard/relay.config.json

```diff
@@ -1,7 +1,7 @@
 {
   "src": "./src",
   "language": "typescript",
-  "schema": "./src/workflows.graphql",
+  "schema": "../relay-workflows-lib/supergraph.graphql",
```

Comment:
This feels a little concerning, is there a better way of passing the schema at build time? If not perhaps we should lift the `supergraph.graphql` file up into the `frontend` dir?

Reply:
Yeah, I was not so sure about this. I think we could either

- copy the schema into multiple subdirectories in the Dockerfile
- move the schema into the frontend dir and point all the relay configs there